model_analyzer/reports/report_manager.py

```diff
@@ -421,7 +421,8 @@ def _build_summary_table(self,
                                         title="Report Table")
 
         sorted_measurements = sorted(self._summary_data[report_key],
-                                     key=lambda x: x[1])
+                                     key=lambda x: x[1],
+                                     reverse=True)
```

Comment:
Can you explain this? Why do we want reverse=True here? Because it sorts ascending by default and we want descending?

Reply:
That's correct - previously, it was relying on the opposite polarity `__lt__`, to sort in the correct direction.